src/util.js

```diff
@@ -118,4 +120,47 @@ function letMixin(Type) {
   return Type;
 }
 
-export { deprecate, serviceResponse, letMixin };
+/**
+ * Utility function for determining if detect-browser package successfully set
+ * itself up and determined the platform
+ */
+function canDeterminePlatform() {
+  return !!platform;
+}
+
+/**
+ * Utility function to get the name of the platform running the code. For a
+ * browser, it will be the browser's name; otherwise, it will be 'node'.
+ */
+function getPlatformName() {
+  return !!platform && platform.name;
+}
+
+/**
+ * Utility function to get the version of the platform running the code.
+ */
+function getPlatformVersion() {
```

Comment:
instead of having separate functions for `canDeterminePlatform`, `getPlatformName`, and `getPlatformVersion` maybe we'd be better with just having one function `getPlatform` that returns an object `{name: "...", version: "...."}` or `null` if theres no data available? I may be misunderstanding the intention of `canDeterminePlatform()`...